packages/backend/src/assets/ai.json

```diff
@@ -254,6 +260,9 @@
       "license": "Apache-2.0",
       "url": "https://huggingface.co/ibm-granite/granite-3.3-8b-instruct-GGUF/resolve/main/granite-3.3-8b-instruct-Q4_K_M.gguf",
       "memory": 4939212390,
+      "properties": {
+        "jinja": "true"
+      },
```

Comment:
Sorry for the noob question.

I'm working on function-calling via MCP on the Playground.
However, in order to make things work, I had to add `'MODEL_JINJA=true'` as an entry to the array defined in the following line:

https://github.com/containers/podman-desktop-extension-ai-lab/blob/9e0e39c01fdf6f6a4957744e8de7736021c44b37/packages/backend/src/workers/provider/LlamaCppPython.ts#L99

How is this property actually consumed?

Reply:
Okay, found it, I assume it's this part here:

https://github.com/containers/podman-desktop-extension-ai-lab/blob/878854ad6ddcc8d8551404b11fd124f595b81e96/packages/backend/src/utils/modelsUtils.ts#L89-L100